### DISCUSSION TITLE
utility classes not added to the build files

### DISCUSSION THREAD

**Discussion by x-o1d** [tailwindlabs/tailwindcss#10280] [Help]
**What version of Tailwind CSS are you using?**

3.1.6

**What build tool (or framework if it abstracts the build tool) are you using?**

NX

**What version of Node.js are you using?**

16.1

**What browser are you using?**

Chrome, Firefox

**What operating system are you using?**

macOS

**Reproduction URL**

The behavior of the build system is unpredictable and erratic and cannot be consistently reproduced. 

**Describe your issue**

classes like ml-auto/rounded-XX are sometimes in the build but sometimes not in the build.

**Comment by thecrypticace** [comment]
Hey, can you give us more info about what tools you are using, how you are using Tailwind CSS (PostCSS plugin, Standlone CLI, npm CLI, etc…), and more of a description of what the behavior it is that you are seeing?

**Comment by aenial51** [comment]
Hello
Utilities are simple HTML classes typically scoped to a single CSS property, like border-style or background-color . Utilities can be used additively to style an object from scratch or to override a style defined in component CSS. Building with CSS utility classes is an incredible boost to productivity and organization. It allows you to define a library of property: value pairs that power all your styles, managed from a single directory.
